daemons/controld/controld_callbacks.c

```diff
@@ -203,9 +211,12 @@ peer_update_callback(enum crm_status_type type, crm_node_t * node, const void *d
                 crm_notice("Our peer connection failed");
                 register_fsa_input(C_CRMD_STATUS_CALLBACK, I_ERROR, NULL);
 
-            } else if (pcmk__str_eq(node->uname, fsa_our_dc, pcmk__str_casei) && crm_is_peer_active(node) == FALSE) {
+            } else if (pcmk__str_eq(node->uname, controld_globals.dc_name,
+                                    pcmk__str_none)
```

Comment:
Node name comparisons should always be case-insensitive, since they are often DNS names, which are case-insensitive.

Reply:
I assumed -- apparently incorrectly -- that all the node names would be coming from the Pacemaker node objects, based on the configuration and not based on DNS or "the actual uname" or anything.

Reply:
It varies too much to do anything but always use case-insensitive. Node names can come from the corosync configuration, the pacemaker configuration, command-line arguments, hostname calls (not a good idea but it happens), etc.